.circleci/cimodel/data/simple/util/docker_constants.py

```diff
@@ -2,7 +2,7 @@
 
 # ARE YOU EDITING THIS NUMBER?  MAKE SURE YOU READ THE GUIDANCE AT THE
 # TOP OF .circleci/config.yml
-DOCKER_IMAGE_TAG = "209062ef-ab58-422a-b295-36c4eed6e906"
+DOCKER_IMAGE_TAG = "fff7795428560442086f7b2bb6004b65245dc11a"
```

Comment:
Hmm, has image tag pattern changed recently?

Reply:
This looks like the new docker image tags introduced in https://github.com/pytorch/pytorch/pull/40194

```
$ git rev-parse HEAD:.circleci/docker
fff7795428560442086f7b2bb6004b65245dc11a
```

These can be used now if wanted.